client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -0,0 +1,213 @@
+import React, { Fragment, PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+
+import { CohortIcon } from 'assets/images/icons';
+import { updateCohort } from '../model/actions';
+import { RouterMatchPropType } from 'common/constants/propTypes';
+
+class CohortHeader extends PureComponent {
```

Comment:
Proposal: This feature will appear in list also. Do we want to have two separate components CohortHeader and ListHeader with almost the same logic? Can we move some common logic to some more general Header component? We could do this while working on ListHeader... but the the will be more to do...

Reply:
It's not so general, because those two areas look different in the list and in the cohort. 
And I don't want to mix it up with list cause list have much more options like private public, and there can be more changes in future in list view. So we would perhaps change this back to ListHeader and Cohortheader in future, because of coming changes and ideas.